backend/src/main/java/de/neuefische/backend/appUser/AppUser.java

```diff
@@ -11,5 +11,7 @@ public class AppUser {
     private String id;
     private String username;
     private String password;
-   // private String iban;
+    private int accountNumber;
+    private String accountType;
+    private double accountBalance;
```

Comment:
Would it be advisable to employ BigDecimals here?

Reply:
Discussed that with a coach before. The advice was to use double.

Reply:
*Check*